Formula/poac.rb

```diff
@@ -0,0 +1,34 @@
+class Poac < Formula
+  desc "Package manager for C++"
+  homepage "https://github.com/poacpm/poac"
+  url "https://github.com/poacpm/poac/archive/refs/tags/0.3.4.tar.gz"
+  sha256 "972727bd1f45f9cb1a0aae1994533fae75234614ffe5df2a316690698db9dd74"
+  license "Apache-2.0"
+  head "https://github.com/poacpm/poac.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on "boost"
+  depends_on macos: :big_sur # C++20
+  depends_on "openssl@3"
+
+  on_linux do
+    depends_on "gcc"
+  end
+  fails_with gcc: "5" # C++20
+
+  def install
+    system "cmake", "-B", "build", *std_cmake_args
+    system "cmake", "--build", "build", "--target", "install"
```

Comment:
```suggestion
    system "cmake", "-S", "." "-B", "build", *std_cmake_args
    system "cmake", "--build", "build"
    system "cmake", "--install", "build"
```
This is the format we use for many other formulae that use CMake, so we should try to use this if it's equivalent (and it looks like it should be).

Reply:
Ah, thank you!